examples/gpt_neox/README.md

```diff
@@ -0,0 +1,43 @@
+# GPT-NeoX on MosaicML Platform (MPlatform)
+
+[The MosaicML Platform (MPlatform)](https://www.mosaicml.com/blog/mosaicml-cloud-demo) enables easy training of distributed machine learning (ML) jobs. In this folder, we provide an example of how to run [GPT-NeoX](https://github.com/EleutherAI/gpt-neox), EleutherAI's library for training large language models, on MPlatform.
```

Comment:
[nit] I've been explicitly told the "Platform" is supposed to be all lowercase 

```suggestion
The [MosaicML platform](https://www.mosaicml.com/blog/mosaicml-cloud-demo) enables easy training of distributed machine learning (ML) jobs. In this folder, we provide an example of how to run [GPT-NeoX](https://github.com/EleutherAI/gpt-neox), EleutherAI's library for training large language models, on the Mosaic platform.
```

Reply:
got it, switched over everywhere